### DISCUSSION TITLE
Products Import/Export

### DISCUSSION THREAD

**Discussion by glennjacobs** [lunarphp/lunar#1025] [Feature Requests]

**Comment by de-raaf-media** [comment]
Is this an available feature?

**Reply by de-raaf-media** [reply]
Bump

**Reply by alecritson** [reply]
Not yet, we're currently looking to get 1.0 alpha release done and then with some updates to Filament we'll take another look at this.

**Comment by lguichard** [comment]
@de-raaf-media That's very simple to create an export on 1.x version with Filament panel. 

Follow up the documentation 1.x : https://docs-v1.lunarphp.io/admin/extending/pages.html

Extend resource list with headerActions and configure Filament's exports batches : https://filamentphp.com/docs/3.x/actions/prebuilt-actions/export: 

```php
 public function headerActions(array $actions): array
    {
        $actions = [
            ...$actions,
            Actions\ExportAction::make()
                ->label(__('admin.actions.export.product.label'))
                ->exporter(ProductExporter::class)
                ->fileDisk(config('filesystems.default')),

            Actions\ImportAction::make()
                ->label(__('admin.actions.import.stock.label'))
                ->importer(ProductStocksImporter::class),
        ];

        return $actions;
    }
```

It's magic, now you have an export button ;-) 

![Capture d’écran 2024-02-14 à 10 34 19](https://github.com/lunarphp/lunar/assets/6489718/cc05400b-5aa9-46a8-9eb3-5fa2febb38d8)

**Reply by de-raaf-media** [reply]
Cool!
